mill-build/src/millbuild/Cs.scala

```diff
@@ -0,0 +1,37 @@
+package millbuild
```

Comment:
May I suggest to use `coursierbuild`, to make it more obvious it's custom stuff for coursier.

Reply:
Should be fixed